cygnus-twitter/src/main/java/com/telefonica/iot/cygnus/sinks/TwitterSink.java

```diff
@@ -0,0 +1,482 @@
+/**
+ * Copyright 2016 Telefonica Investigación y Desarrollo, S.A.U
```

Comment:
In the case this class remains, I think we should try to find the way of using a unique class instead of having `NGSISink`, `TwitterSink`, and many other thar may come in the future. I'm saying this because it seems `TwitterSink` is equals to `NGSISink` except for it creates batches of twits instead of batches of NGSI notifications. I think this could be "easily" fixed by creating a parameterizable generic `Batch<Type>` class or soemthing like that. I'll create an issue in order to discuss this outside this review :)